contracts/src/main/kotlin/com/r3/developers/apples/states/BasketOfApples.kt

```diff
@@ -0,0 +1,22 @@
+package com.r3.developers.apples.states
+
+import com.r3.developers.apples.contracts.BasketOfApplesContract
+import net.corda.v5.ledger.utxo.BelongsToContract
+import net.corda.v5.ledger.utxo.ContractState
+import java.security.PublicKey
+
+@BelongsToContract(BasketOfApplesContract::class)
+class BasketOfApples(
+    val description: String,
+    val farm: PublicKey,
+    val owner: PublicKey,
+    val weight: Int,
+    private val participants: List<PublicKey>
```

Comment:
As per previous comment...

Reply:
Captured and will be done in https://r3-cev.atlassian.net/browse/CORE-15314 and https://r3-cev.atlassian.net/browse/CORE-15315